### ISSUE TITLE
Reference to drying solvents - wrong ref?

### ISSUE THREAD

**Issue by mattodd** [opensourcecatalysis/OPS-Paper1#37]
In "Materials and Methods" ref 1 (the original PS paper) is used as the lit ref for drying of solvents. I suspect this is not the right reference. Could someone fix, replace and close this issue?

**Comment by paulking1995** [comment]
I believe the correct reference should be:

Williams, D. B. G.; Lawton, M. Drying of Organic Solvents: Quantitative
Evaluation of the Efficiency of Several Desiccants. J. Org. Chem. 2010, 75,
8351–8354, DOI: 10.1021/jo101589h.

HOWEVER, I don't think we really had used the exact same method for activating sieves (I dried them at 300C which is standard) but I believe all of us would have stored solvent over them for at least 1 day so I have changed accordingly. If you disagree, then feel free to dispute. @katb @edwintse
